pom.xml

```diff
@@ -178,11 +178,6 @@
             <scope>test</scope>
         </dependency>
 
-        <dependency>
```

Comment:
Why did you remove this dependency?

Reply:
Because it was duplicated

Reply:
I saw it in the report and that it was duplicated. It is in line 158 but maybe I made a mistake removing it

Reply:
If it's duplicated then it's ok. Thanks